catalog-item.html

```diff
@@ -131,6 +120,7 @@ <h1>ITEM DETAIL</h1>
             <div class="item-info">
                 <h2>sKISIRS#4587</h2>
                 <p>As the first hero of the Neoh Universe, collection has over 9,999 unique skins drawn from the different missions and challenges he faced throughout his life.</p>
+                
                 <div class="item-info-table">
                     <table class="meta-info-item">
```

Comment:
is not suitable as a table because a table is a way of structuring data. It represents the distribution of data into uniform rows and columns. There is no relationship here indicating that a row or column contains related data.

please use dl, dt, dd here

Reply:
`<table class="table-item-pice">` is not a table too

Reply:
In this case, I was thinking along these lines-- 
Since the number of products on the site can change, as well as their characteristics (e.g. price) . I assumed that all information is taken from a database (e.g. table) outside the site.
From this I concluded that it can be done with AJAX and JS. But we have not yet had time to pass them, and I do not have enough knowledge to correctly display it in html code.